source/plot/LinePlotData2D.ooc

```diff
@@ -12,30 +12,42 @@ LineStyle: enum {
 
 LinePlotData2D: class extends PlotData2D {
 	lineStyle: LineStyle { get set }
+	dataSeries: VectorList<FloatPoint2D> { get set }
```

Comment:
Unless I need the specific functionality, this is easier to replace with another type as I did with free shapes.